Sources/PointFree/Emails/ChangeEmailConfirmation.swift

```diff
@@ -0,0 +1,50 @@
+import Html
+import HtmlCssSupport
+import Optics
+import Prelude
+import Styleguide
+
+let confirmEmailChangeEmailView = simpleEmailLayout(confirmEmailChangeEmailBody)
+  .contramap { user, newEmailAddress in
+    SimpleEmailLayoutData(
+      user: user,
+      newsletter: nil,
+      title: "Email change confirmation",
+      preheader: "",
+      data: (user, newEmailAddress)
+    )
+}
+
+private let confirmEmailChangeEmailBody = View<(Database.User, EmailAddress)> { user, newEmailAddress in
+  emailTable([style(contentTableStyles)], [
+    tr([
+      td([valign(.top)], [
+        div([`class`([Class.padding([.mobile: [.all: 2]])])], [
+          h3([`class`([Class.pf.type.title3])], ["Confirm email change"]),
+          p([`class`([Class.padding([.mobile: [.topBottom: 2]])])], [
+            "We received a request to change your email on Point-Free. Your current email is ",
+            span([`class`([Class.type.semiBold])], [.text(encode(user.email.unwrap))]),
+            ", and the new email is ",
+            span([`class`([Class.type.semiBold])], [.text(encode(newEmailAddress.unwrap))]),
+            ". If you want to make this change, just click the confirmation link below:"
+            ]),
+
+          p([`class`([Class.padding([.mobile: [.top: 2, .bottom: 3]])])], [
+            a(
+              [ href(url(to: .confirmEmailChange(userId: user.id, emailAddress: newEmailAddress))),
```

Comment:
very cool that we can link to routes so easily like this, but under the hood it's encrypting the payload and putting it into a query param!